### DISCUSSION TITLE
could not connect to server：read ECONNRESET， how to fix it?

### DISCUSSION THREAD

**Discussion by aresbit** [community/community#11960] [Copilot Conversations]
[ERROR] [default] [2022-02-23T11:03:56.400Z] GitHub Copilot could not connect to server. Extension activation failed: "read ECONNRESET"

**Comment by davecheney** [comment]
Is it possible that the GFW is blocking your connections?

**Reply by aresbit** [reply]
I donot know,  how can fix it ?  use VPN?

**Reply by aresbit** [reply]
after using VPN , yet cannot works. emmm...

**Reply by luxuncang** [reply]
I have encountered similar problems recently. How did you solve them？

**Reply by luxuncang** [reply]
To a large extent, it is a network problem. The hosts file is modified to solve it.
```192.30.255.116 api.github.com```

**Reply by aresbit** [reply]
I modified  hosts file , but yet not work well, sad

**Reply by aresbit** [reply]
谢回复，或许是GFW，我提提issue后修改hosts，再换美区科学登录就解决了。

**Reply by ZhibangYue** [reply]
I have modified hosts and connected VPN, but it doesn't work.

**Reply by Nuevo009** [reply]
same. It happened after I went back to school and my ISP changed from China Unicom to CERNET.

Tried the method mentioned above but none of these works.

Finally, I use my mobile hotspot and the ISP changed to China Mobile. Copilot just works fine.

**Reply by Nuevo009** [reply]
我觉得并不是 GFW 的问题 @aresbit @luxuncang @ZhibangYue 你们的 ISP 是不是也是 CERNET ? 因为这个 discussion 正好发在学生返校的节点上（

**Reply by Nuevo009** [reply]
我正在和我学校这的信息办沟通，另外我用测试 GH 节点的小工具测试的结果是，连接校园网的情况下任何节点都会 connect time out 或者 certificate verification failed。其他三网则正常

**Reply by ZhibangYue** [reply]
@Nuevo009 You're right! Thanks! It was CERNET that caused the error.

**Reply by Nuevo009** [reply]
@ZhibangYue 

如果你们学校的校园网有其他 ISP 的出口的话，建议让信息办老师把 GitHub 解析到的常见 ip 换到其他出口上。

**Reply by wensc1996** [reply]
1.23有问题 回溯到1.22解决了

**Reply by fei550815954** [reply]
me too
